src/components/steps/general-info/Funders.tsx

```diff
@@ -92,18 +92,13 @@ const Funders: React.FC<FundersProps> = ({ project, onChange }) => {
                     cancelText={i18n.t("No")}
                     saveText={i18n.t("Yes")}
                     onCancel={() => {
-                        setAutomaticFill(false);
                         setDialogOpen(false);
                     }}
                     onSave={() => {
-                        setAutomaticFill(true);
-                        if (automaticFill) {
-                            const additionalDesignation = value.reduce(
-                                (accumulator, { shortName }) =>
-                                    accumulator ? `${accumulator}-${shortName}` : shortName,
-                                ""
-                            );
-                            updateAdditionalDesignationField("additional", additionalDesignation);
+                        if (fundersAdded) {
+                            code.push(fundersAdded[fundersAdded.length - 1].shortName);
```

Comment:
As stated above, simply use `project.funders` and concatenate codes with join, no need for state.